consensus/istanbul/protocol.go

```diff
@@ -29,9 +29,18 @@ const (
 // protocolName is the official short name of the protocol used during capability negotiation.
 const ProtocolName = "istanbul"
 
-// ProtocolVersions are the supported versions of the eth protocol (first is primary).
+// ProtocolVersions are the supported versions of the istanbul protocol (first is primary).
```

Comment:
i don't think primary refers to which is the current one. The current is always the biggest version number. I would remove the docs

Reply:
I was confused by it it, at first I thought it was something we added, but then I saw it's not.  It's also present on the LES protocols, though there when they added version 3 they put it last not first.  I assume that was a mistake, and that the intention is to always place the highest version first, because in some places in the code it references using `[0]`. e.g. https://github.com/celo-org/celo-blockchain/blob/master/eth/backend.go#L559 , but there are some other places.